spec/image_spec.rb

```diff
@@ -95,72 +81,117 @@
 
     context 'when given a bitonal tif' do
       before do
-        # Need to force group4 compression to get ImageMagick to create bitonal tiff
-        generate_test_image(TEST_TIF_INPUT_FILE, image_type: 'Bilevel', compress: 'group4')
+        generate_test_image(TEST_TIF_INPUT_FILE, color: 'bin', bands: 1, depth: 1)
       end
 
       it 'creates grayscale jp2' do
         expect(File).to exist TEST_TIF_INPUT_FILE
         expect(File).not_to exist TEST_JP2_OUTPUT_FILE
+        expect(assembly_image.exif.samplesperpixel).to be 1
+        expect(assembly_image.exif.bitspersample).to be 1
+        expect(assembly_image).not_to have_color_profile
         result = assembly_image.create_jp2(output: TEST_JP2_OUTPUT_FILE)
+        expect(result).to be_a_kind_of described_class
+        expect(result.path).to eq TEST_JP2_OUTPUT_FILE
         expect(TEST_JP2_OUTPUT_FILE).to be_a_jp2
-        expect(result.exif.colorspace).to eq 'Grayscale'
+        # expect(result.exif.colorspace).to eq 'Grayscale'
       end
```

Comment:
@calavano indeed this expectation fails now and was commented out in your test PR as well...was this expected or do we have a problem to deal with?

Reply:
Looking at my code again, the temporary tiff is being normalized to color. We would need to add an additional clause in the case statement to account for it. Since the jp2 is only a display derivative, I don't feel this would be a problem. Having jp2s created in grayscale again would need it enabled.

Reply:
Can you say more about what this means?  Do we need to make some adjustments to the `generate_test_image` method in spec helper?

Reply:
I am confused also:   Do we never want grayscale jp2 images now, or do we need to add a test (and test file and/or code) creating grayscale jp2 when indicated?

Reply:
We sorted this out, it's not an issue anymore in the latest code for this PR...though there is still one issue to investigated (noted in the PR description).

Reply:
@peetucket would you please make a ticket for the remaining issue and either assign it to Tony or tag him?  Thank you.